### ISSUE TITLE
Nothing happens if we call fitBounds, flyTo, setCamera 

### ISSUE THREAD

**Issue by fernandoamz** [rnmapbox/maps#2130]
## Environment

- Mapbox (GL) implementation: [e.g. v10,  rnmapbox, MapboxGL]
- Mapbox (GL) version: [8.6.0]
- @rnmapbox/maps Version [8.5.0]
- Platform: [Android, iOS]
- React Native Version [0.66.0]
- Platform OS: [Android 11, above iOS 15.4]
- Device: [Iphone 13 Pro Max]
- Emulator/ Simulator: Yes
- Dev OS: [OSX 12.5]

## Standalone component to reproduce

```tsx

 const bboxCoords = bbox(shape)

  try {
    cameraRef.current?.fitBounds(
      [bboxCoords[2], bboxCoords[3]],
      [bboxCoords[0], bboxCoords[1]],
      64,
      0
    )
  } catch (e) {
    console.log(e)
  }

```

## Observed behavior and steps to reproduce

We identified this issue because the map is not doing zoom called by the fitBounds, currently, this is working on versions under `iOS 15.5` good, But we identified this issue when the QA team did testing on `iOS 15.5`, also I tested on the different simulator with versions above `iOS 15.4` and the app is doing the error, we would like to know a workaround to fix this because our UX is being affected for the issue.

## Expected behavior

As I described above, if we open a map and we call the fitBounds this should do an auto zoom with a version above `iOS 15.4`. 

## Notes / preliminary analysis

I would like to help with this issue if you have a slack, discord, or some forum to ask questions.

## Additional links and references

I've seen that this issue and similar are reported.

https://github.com/rnmapbox/maps/issues/2121
https://github.com/rnmapbox/maps/issues/2001

Thanks && Regards for reading me! I hope you can help me, team! bless!

**Comment by mfazekas** [comment]
Fixed by: #2129

**Comment by fernandoamz** [comment]
Hi @mfazekas, thanks for your quick response, awesome!

Just a quick question: 
Where do I can see the version to get these changes and test by my side?
